pwa-devdocs/src/peregrine/reference/targets/wrappable-talons/index.md

```diff
@@ -5,23 +5,25 @@ title: Wrappable Peregrine Talons
 ## Usage
 
 An extension may need to modify or decorate the behavior of a Peregrine talon.
-Third-party code may need to log to an external source when certain events occur, or may need to extend the model returned by the talon when a component calls it.
+For example, third-party code may need to log to an external source when certain events occur or it may need to extend the model returned by the talon when a component calls it.
+To do this, you can wrap Peregrine talons to extend their functionality.
 
 ### Wrapper modules
 
 Talons are React hooks, and React hooks are plain functions.
 Therefore, you can use the function wrapper pattern to intercept talon functions.
 
-The `talon.wrapWith(module)` method is simlar to the [interceptor pattern](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/plugins.html) also used in Magento backend plugins.
-Peregrine will dynamically inject the code from the passed `module` "around" the implementation of a talon, by passing the talon function through the wrapper function before exporting it.
+The `talon.wrapWith(module)` method is similar to the [interceptor pattern](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/plugins.html) used in Magento backend plugins.
+During the build process, the extensibility framework dynamically wraps a talon implentation with code from a wrapper module by passing the talon function through the wrapper module before exporting it.
```

Comment:
The extensibility framework dynamically wraps a talon implementation with code from a wrapper module by passing the talon function through the wrapper module before exporting it during the build process.

Reply:
Fixed the mispelled word but kept the ordering because it helps to introduce the context at the beginning of the sentence.